util/tracing/Makefile

```diff
@@ -30,8 +31,10 @@ install: trace_to_csv trace_to_chrome trace_to_influxdb
 	mv trace_to_csv $(INSTALL_PREFIX)
 	mv trace_to_chrome $(INSTALL_PREFIX)
 	mv trace_to_influxdb $(INSTALL_PREFIX)
-	ln -f -s launch-fedsd.sh $(INSTALL_PREFIX)/fedsd
-	chmod +x launch-fedsd.sh
+	cp ./visualization/fedsd.py $(INSTALL_PREFIX)
```

Comment:
Do we want to maintain the `.py` extension after installation?

Reply:
It is not maintained. In line 35, a symbolic link is created with the name `fedsd`.

Reply:
OK, I see what you're trying to do. But if `fedsd` is on the path, then so is `fedsd.py`. Is that the intention?
I would imagine we would want something like:
```
        mkdir -p $(INSTALL_PREFIX)/lib/fedsd
        cp ./visualization/fedsd.py $(INSTALL_PREFIX)/lib/fedsd
	ln -s $(INSTALL_PREFIX)/lib/fedsd/fedsd.py $(INSTALL_PREFIX)/bin/fedsd
        chmod +x $(INSTALL_PREFIX)/bin/fedsd
```